internal/scheduler/jobs/patch_anchor_platform_transactions_job.go

```diff
@@ -45,7 +46,16 @@ func (j patchAnchorPlatformTransactionsCompletionJob) GetInterval() time.Duratio
 }
 
 func (j patchAnchorPlatformTransactionsCompletionJob) Execute(ctx context.Context) error {
-	err := j.service.PatchAPTransactionsForPayments(ctx)
+	t, err := tenant.GetTenantFromContext(ctx)
+	if err != nil {
+		return fmt.Errorf("error getting tenant from context for %s: %w", paymentFromSubmitterJobName, err)
+	}
+	if !t.DistributionAccountType.IsStellar() {
+		log.Ctx(ctx).Debugf("Skipping job %s for tenant %s as it uses a %s Distribution account", j.GetName(), t.ID, t.DistributionAccountType.Platform())
+		return nil
+	}
+
+	err = j.service.PatchAPTransactionsForPayments(ctx)
```

Comment:
TBH I don't think we should be completely killing this call when we're not on Stellar.

Can we add a ticket to re-add it after further investigation?

Reply:
I re-enabled this job, and I fixed the issue causing this to fail when a payment is failed (nil stellar_transaction_id). 

I also created this ticket to re-evaluate this job https://stellarorg.atlassian.net/browse/SDP-1277